students/kosarzewski_maciej/lesson_04_unit_testing/collatz/collatz_sequence.py

```diff
@@ -0,0 +1,17 @@
+def collatz(number):
+    if number <= 0:
+        raise ValueError
+    elif number % 2 == 0:
```

Comment:
elif is not necessary here. Please try to avoid code duplication

def collatz(number):
    if number <= 0:
        raise ValueError
    if number % 2 == 0:
        value = number / 2
    else:
        value = (3 * number) + 1
    print(value)
    return value